Test/TestData/2021-06-08-19-48-32/Task Settings/Task1-SearchTaskconfig.toml

```diff
@@ -0,0 +1,88 @@
+TaskType = "Search"
```

Comment:
i think this file should not be included?

Reply:
same with the file "Test/TestData/2021-06-08-19-48-32/Task1-SearchTask/model.zip"